docs/shared/concepts/inferred-targets.md

```diff
@@ -0,0 +1,88 @@
+
+Inferred targets are created with the following principles in mind:
+
+- As much as possible, tooling configuration stays in the tool configuration files.  
+  i.e. There should not be executor options that duplicate the tooling configuration.
+- Running an inferred target should be identical to launching that tool in the project directory.  
+  i.e. Running `nx lint my-project` should be the same as running `eslint .` from the `my-project` directory.
+- Inferred options (such as `inputs` and `outputs`) should be calculated from the tooling config.
+- Default set up should have very little nx-specific config
+- All default settings can be overwritten in the project configuration (`package.json` or `project.json`)
+
+## Setup Inferred Targets
+
+To start using inferred targets, you need to:
+
+1. Install the plugin
+
+   ```shell
+   npm i -D @nx/eslint
+   ```
+
+2. Enable inferred targets in the `nx.json` file
+
+   ```json {% fileName="nx.json" %}
+   {
+     "plugins": [
+       {
+         "plugin": "@nx/eslint/plugin",
+         "options": {
+           "targetName": "lint"
+         }
+       }
+     ]
+   }
+   ```
+
+## How Exactly Does a Plugin Create an Inferred Target?
+
+Every plugin has its own custom logic, but in order to infer targets, they all go through the following steps.
+
+### 1. Detect Tooling Config in Projects
+
+The plugin will check each project in your repo for tooling configuration files. The `@nx/eslint` plugin checks for `.eslintrc.json` files at the root of the project. If the configuration files are found, targets are inferred for that project.
+
+### 2. Create an Inferred Target
+
+The plugin will create a target for you that will use the tooling config that was detected. The name for the target (i.e. `lint`) is set in the `options` for the plugin in `nx.json`.
+
+### 3. Set Inferred Target Options Based on Tooling Config
+
+The plugin will automatically set options for the inferred targets. This includes, at a minimum, `inputs` and `outputs`.
+
+## Configuration Precedence
+
+As much as possible, tooling configuration stays in the tooling configuration files themselves. For nx-specific target configuration, there are three possible sources:
+
+- `targetDefaults` in the `nx.json` file
+- Inferred values from plugins
+- Project-specific configuration in `package.json` or `project.json`
+
+The target configuration is calculated in that order. Project-specific configuration overwrites inferred values, which overwrite the `targetDefaults`.
+
+## Debug Inferred Targets
+
+To view the target settings that were inferred for a project, show the project details either from the command line or using Nx Console.
+
+```shell
+nx show project my-project --web
```

Comment:
I also added [a task](https://linear.app/nxdev/issue/NXP-395/nx-show-project-improvements) to perhaps have a better view of targets without the project view.

Maybe something like

```shell
$ nx show project demo
root: apps/demo
projectType: application
tags: type:app, scope:foo
targets:
- build (source: @nx/vite/plugin)
- test (source: @nx/vite/plugin)
- serve (source: @nx/vite/plugin)
- lint (source: @nx/eslint/plugin)
- foo (source: apps/demo/project.json)

To view raw JSON output, pass the --json option.
```

Seems weird to require a web UI in order to show project info. Thoughts?

Reply:
Yes, I'm expecting this section to change as the view gets updated.